docs/doc/14-sql-commands/00-ddl/103-aggregating-index/create-aggregating-index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: CREATE AGGREGATING INDEX
+---
+
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.151"/>
+
+import EEFeature from '@site/src/components/EEFeature';
+
+<EEFeature featureName='AGGREGATING INDEX'/>
+
+Creates a new aggregating index in Databend.
+
+## Syntax
+
+```sql
+CREATE AGGREGATING INDEX <index_name> AS SELECT ...
+```
+
+- When creating aggregating indexes, limit their usage to standard aggregate functions (e.g., AVG, SUM, MIN, MAX, COUNT), while keeping in mind that GROUPING SETS, window functions, LIMIT, and ORDER BY are not accepted.
+
+- The query filter scope defined when creating aggregating indexes should either match or encompass the scope of your actual queries.
+
+- Aggregating indexes only work for single-table queries. To confirm if an aggregating index works for a query, use the [EXPLAIN](../../90-explain-cmds/explain.md) command to analyze the query.
```

Comment:
Aggregating index may match with the child plan of a multi-table query's plan. See https://github.com/datafuselabs/databend/blob/1c9063761b8bcc55074829b02ec97f5bd25e24fd/tests/sqllogictests/suites/mode/standalone/ee/explain#L161